SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -404,6 +391,38 @@ void CScalarSolver<VariableType>::ImplicitEuler_Iteration(CGeometry* geometry, C
   CompleteImplicitIteration(geometry, solver_container, config);
 }
 
+template <class VariableType>
+void CScalarSolver<VariableType>::ExplicitEuler_Iteration(CGeometry* geometry, CSolver** solver_container,
+                                                          CConfig* config) {
+
+  const auto flowNodes = solver_container[FLOW_SOL]->GetNodes();
```

Comment:
You can (and maybe should) apply clang-format to all Scalar-related files as they (should) adhere to theses rules already and thus I dont expect  any (too many) additional changes. (I know you didn't because clang-format does not make a newline in between here 🕵️ )

But for other files I would maybe be careful how many changes will be introduced

Reply:
done in bb09595